sources/packages/backend/apps/db-migrations/src/sql/Queue/Add-assessment-workflow-queue-retry.sql

```diff
@@ -0,0 +1,14 @@
+INSERT INTO
+    sims.queue_configurations(queue_name, queue_configuration)
+VALUES
+    (
+        'assessment-workflow-queue-retry',
+        '{
+            "cron": "0 0 * * * ?",
```

Comment:
usually, the cron expression has only 5 places, right? correct me if Iam wrong, If Iam wrong, how often does the scheduler run?

Reply:
About the cron expression fields: on [Bull documentation](https://docs.bullmq.io/guide/jobs/repeatable) it mentions that it uses [cron-parser lib](https://www.npmjs.com/package/cron-parser) that can receive 5 or 6 params.

On https://www.freeformatter.com/cron-expression-generator-quartz.html there's a a tool to describe the cron expression:
![image](https://github.com/bcgov/SIMS/assets/78114138/b8f621bf-ec36-453b-85d6-affa2e878d1a)